test/PipeSpecTest.php

```diff
@@ -33,7 +31,7 @@ public function process(
         $spec       = new PipeSpec('container_key_string', $middleware, 'another_key');
         self::assertSame(
             ['container_key_string', $middleware, 'another_key'],
-            array_values($spec->getSpec())
```

Comment:
array_values was used to make sure it is not affected by different keys by discarding keys

Reply:
Not needed anymore, implementation can't receive keys in input.

Reply:
added back with `@psalm-suppress RedundantCast` https://github.com/laminas/laminas-mvc-middleware/pull/22/commits/f30ad4c8bf0cbb348810f5570e7432e469f19c1d

Reply:
ok, removed again https://github.com/laminas/laminas-mvc-middleware/pull/22/commits/93ff4a47d587b0681ad7e5d49b288445fb172b04